drivers/ce_performance/app/models/ce_performance/report.rb

```diff
@@ -486,7 +486,7 @@ def detail_headers(key: nil) # rubocop:disable Lint/UnusedMethodArgument
       @detail_headers ||= {}.tap do |headers|
         headers.merge!(
           {
-            'client_id' => 'Warehouse Client ID',
+            'client_id' => 'Warehouse Source Client ID',
```

Comment:
Do we want to update this report to include the destination ID? Would require a migration

Reply:
Yes, we should do this like you did for the PATH report.